2020/day-1-report-repair/index.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { findSumParts } from './'
+
+describe('Day 1: Report Repair', () => {
+  it('Finds a number pair whose sum equals 2020, then multiplies those two numbers', () => {
+    const numbers = [1721, 979, 366, 299, 675, 1456]
+    const [a, b] = findSumParts(2020, numbers)
+    const expected = 514579
+
+    expect(a * b).toBe(expected)
```

Comment:
I actually feel like this should be a separate test. So we have one test for checking the numbers returned are what are expected. And we have another test to check the calculation WDYT?

Reply:
Wouldn't that second test just be testing the multiplication of two numbers? Basically testing that `*` does what it should?